sa/sa.go

```diff
@@ -267,18 +267,19 @@ func (ssa *SQLStorageAuthority) AddPrecertificate(ctx context.Context, req *sapb
 		// would be a misissuance and miscalculating whether the cert is a renewal or
 		// not for the purpose of rate limiting is the least of our troubles.
 		isRenewal, err := ssa.checkFQDNSetExists(
+			ctx,
 			txWithCtx.SelectOne,
 			parsed.DNSNames)
 		if err != nil {
 			return nil, err
 		}
 
-		err = addIssuedNames(txWithCtx, parsed, isRenewal)
+		err = addIssuedNames(ctx, txWithCtx, parsed, isRenewal)
```

Comment:
Feels weird to pass both `ctx` and `txWithCtx` in the same argument list. It raises questions in the reader's mind about which context will actually be respected, if they're different. Can we just rename `txWithCtx` to `tx`, since there's no such thing as a transaction _without_ a context?

Reply:
Ah right, I forgot this step! Yeah I wanted to rename all `txWithCtx` to `tx`.